pkgs/top-level/aliases.nix

```
@@ -570,6 +570,7 @@ mapAliases ({
   openisns = open-isns; # added 2020-01-28
   openjpeg_1 = throw "openjpeg_1 has been removed, use openjpeg_2 instead"; # added 2021-01-24
   openjpeg_2 = openjpeg; # added 2021-01-25
+  openmtp123 = libopenmpt; # added 2021-09-05
```

Sorry I missed this.

```suggestion
  openmpt123 = libopenmpt; # added 2021-09-05
```